### PR TITLE
[MHV-56098, MHV-55648, MHV-56084] V3 Component Fixes - SM

### PR DESCRIPTION
**Pull request by vsaleem**
## Summary

- Removed ALT text from 'unread message' icon
- Adjusted pagination length to 7 on MessageList pages
- Fixed text wrapping on Attachment Name when adding new files for attachments
- These changes were made to conform to the latest V3 uswds 38.0 design system library updates.

- Manual Testing
_Note: This field is mandatory for UI changes (non-component work should NOT have screenshots)._

|         | Before | After |
| ------- | ------ | ----- |
| Mobile  |image|image|
| Desktop |        |       |

## What areas of the site does it impact?

*Va.gov - Secure Messaging*

### Quality Assurance & Testing

- [ ] I fixed|updated|added unit tests and integration tests for each feature (if applicable).
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs
- [ ] Linting warnings have been addressed
- [ ] Documentation has been updated ([link to documentation](#) \*if necessary)
- [ ] Screenshot of the developed feature is added
- [ ] [Accessibility testing](https://depo-platform-documentation.scrollhelp.site/developer-docs/wcag-2-1-success-criteria-and-foundational-testing) has been performed

### Error Handling

- [ ] Browser console contains no warnings or errors.
- [ ] Events are being sent to the appropriate logging solution
- [ ] Feature/bug has a monitor built into Datadog or Grafana (if applicable)

### Authentication

- [ ] Did you login to a local build and verify all authenticated routes work as expected with a test user

### :warning: Team Sites (only applies to modifications made to the VA.gov header) :warning:

- [ ] The vets-website header does not contain any web-components
- [ ] I used the [proxy-rewrite steps](https://github.com/department-of-veterans-affairs/vets-website/tree/main/src/applications/proxy-rewrite#local-dev) to test the injected header scenario
- [ ] I reached out in the `#sitewide-public-websites` Slack channel for questions

## Requested Feedback

(OPTIONAL) _What should the reviewers know in addition to the above. Is there anything specific you wish the reviewer to assist with. Do you have any concerns with this PR, why?_

### CODE REVIEW

File: src/applications/mhv/secure-messaging/components/ThreadList/ThreadListItem.jsx [hunk 1]
```diff
@@ -92,11 +92,10 @@ const ThreadListItem = props => {
           (unreadMessages && (
             <span>
               <i
-                role="img"
                 aria-hidden="true"
                 className="unread-icon vads-u-margin-right--1 vads-u-color--primary-darker fas fa-solid fa-circle"
                 data-testid="thread-list-unread-icon"
-                alt="Unread message icon"
+                alt=""
```
Comment (oleksii-morgun) [inline src/applications/mhv/secure-messaging/components/ThreadList/ThreadListItem.jsx:98]: Should we consider just removing this attribute if it is not used?

**Review by alex-morgun-1 — DISMISSED** [review]
Looks great 👍🏻 Just one comment re: the `alt` prop, but I will leave it up to you to determine. Otherwise, approved ✅

**Review by KyleCardwell — DISMISSED** [review]
Looks good!  (Same question about the empty alt prop)

**Review by rmessina1010 — DISMISSED** [review]
LGTM

**Comment by vsaleem** [comment]
The `alt` tag was removed - got confirmation from a11y that `alt` is not needed since we are using aria-hidden in this component. @oleksii-morgun @KyleCardwell

**Review by alex-morgun-1 — CHANGES_REQUESTED** [review]
since the new `src/applications/mhv-secure-messaging` directory is up, let's migrate this code over to the new directory

**Review by KyleCardwell — DISMISSED** [review]
Looks good!

File: src/applications/mhv/secure-messaging/components/AttachmentsList.jsx [hunk 2]
```diff
@@ -183,7 +183,7 @@ const AttachmentsList = props => {
                       alt="Attachment icon"
                       aria-hidden="true"
                     />
-                    <span>{file.name} </span>(
+                    <span className="attachment-name-text">{file.name}</span>(
```
Comment (oleksii-morgun) [inline src/applications/mhv/secure-messaging/components/AttachmentsList.jsx:186]: I would suggest not to include these old dir changes whatsoever. They may break cypress tests (as tests are not updated in the old dir) and cause more confusion

File: src/applications/mhv/secure-messaging/components/ThreadList/ThreadsList.jsx [hunk 3]
```diff
@@ -89,13 +89,22 @@ const ThreadsList = props => {
...
+                  onPageSelect={e => paginationCallback(e.detail.page)}
+                  page={pageNum}
+                  pages={Math.ceil(
+                    threadList[0]?.threadPageSize / threadsPerPage,
+                  )}
+                  uswds
+                />
+              </div>
+            </div>
+          </div>
```
Comment (oleksii-morgun) [inline src/applications/mhv/secure-messaging/components/ThreadList/ThreadsList.jsx:107]: same here as above

**Review by alex-morgun-1 — APPROVED** [review]
Looks good 👍🏻

**Pull request merged**